config/personality/base.ini

```diff
@@ -12,3 +12,15 @@ apps_del = ${apps_add_defaults}
 [flatpak-remote-flathub]
 # Delete all default additions and explicitly build the list from scratch.
 apps_del = ${apps_add_defaults}
+apps_add =
+  org.learningequality.Kolibri
```

Comment:
This is probably a fine start but I do wonder if we should actually be putting this into `apps_add_mandatory` in `config/defaults.ini`. It is at least as important a part of Endless OS 4 and beyond as Hack is, IMO.